contracts/trade-shield-contract/src/entry_point/migrate.rs

```diff
@@ -20,59 +10,5 @@ pub fn migrate(
     if msg.account_history_address.is_some() {
         ACCOUNT_HISTORY_ADDRESS.save(deps.storage, &msg.account_history_address)?;
     }
-
-    let admin = "elys16xffmfa6k45j340cx5zyp66lqvuw62a0neaa7w".to_string();
-    PARAMS_ADMIN.save(deps.storage, &admin)?;
-
-    let state = true;
-    let limit_process_order: Option<u128> = Some(100);
-    STAKE_ENABLED.save(deps.storage, &state)?;
-    MARKET_ORDER_ENABLED.save(deps.storage, &state)?;
-    SWAP_ENABLED.save(deps.storage, &state)?;
-    PROCESS_ORDERS_ENABLED.save(deps.storage, &state)?;
-    PERPETUAL_ENABLED.save(deps.storage, &state)?;
-    REWARD_ENABLED.save(deps.storage, &state)?;
-    LEVERAGE_ENABLED.save(deps.storage, &state)?;
-    LIMIT_PROCESS_ORDER.save(deps.storage, &limit_process_order)?;
```

Comment:
@amityadav0 those feature flags params should remain to allow to switch on/off any features avaialble in the contract